.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 install:
-- echo "Skipping install step"
+  - sudo apt-get -y install awscli
```

Comment:
In the other pipelines we use pip to install `awscli`. I don't know what's best, but did you choose this for another reason?

Reply:
Didn't know that, will change! Better to be consistent :)